demo/src/gatsby-theme-teranova/pages/index.mdx

```diff
@@ -9,4 +10,6 @@ import Contact from 'gatsby-theme-teranova/src/components/contact';
 
 <Services />
 
+<Custom />
```

Comment:
Maybe the component should be commented out and the user can just uncomment it if they want to view how a custom component looks, what do you think?

Reply:
This would work for the page content. The navigation would show the content. The section would have to be commented out there as well.

Reply:
I think commenting those two parts and mentioning it in the docs will be okay for now and maybe we could do something else later.

Reply:
Sounds reasonable. I adjusted the demo source code accordingly.